### ISSUE TITLE
Enable Ctrl Alt and Delete command.

### ISSUE THREAD

**Issue by deepeshpradhan** [Microsoft/azserialconsole#61]
While I was doing in-place upgrade to latest Operating system. I was successfully able to until there was prompt for Ctrl Alt and Delete. I think if we have that option from Serial Console it would be usefully for many other cases

**Comment by asinn826** [comment]
@deepeshpradhan, thanks for the feedback! Out of curiosity, what are you trying to do? As context - the Ctrl+Alt+Delete button is hardcoded pretty deep into Windows to always send the signal to the local computer, so I don't think it's possible to get a Windows VM to respond to that via serial console. SAC also might not support that key combination either. If you're trying to reboot the VM, you can use the restart VM button or send an NMI instead.

**Comment by deepeshpradhan** [comment]
@asinn826 

For Testing purpose I was trying to do in-place upgrade though its not supported in Azure. But somehow with workaround mounted OS iso and run upgrade command. OS was installed and was stuck in this prompt. From Boot Diagnostics tab in Azure. I saw it was waiting for my input for CTRL ALT and DEL
![bootdiag](https://user-images.githubusercontent.com/44276115/52588441-594b8300-2df1-11e9-8826-b8e08322724c.JPG)


From Serail Console if we can send Restart VM, NMI or Function command. We can check for option Ctrl+Alt+del key combination

**Comment by asinn826** [comment]
@deepeshpradhan I see what you mean. I know Ctrl+Alt+End is an accepted keystroke for a Ctrl+Alt+Delete alternative when using RDP. I don't think it worked for me in SAC when I tried, but you could give that a shot. If that doesn't work, I don't think there's anything else that can be done to plumb a Ctrl+Alt+Delete command to the VM. I will do some more digging around to confirm.

**Comment by deepeshpradhan** [comment]
@asinn826 Ctrl+Alt+end work only with RDP. 

Actually we should have options out here

![image](https://user-images.githubusercontent.com/44276115/52609745-3346d280-2e33-11e9-93e2-135eadc4064b.png)

**Comment by asinn826** [comment]
@deepeshpradhan Fair enough, and thanks again for raising this! This is in our backlog now.

**Comment by deepeshpradhan** [comment]
@asinn826  Looking forward for this or similar kind of feature. Thank you. Always up for smart enhancement

**Comment by asinn826** [comment]
@CraigWiand fyi

**Comment by asinn826** [comment]
@deepeshpradhan can you try restarting your VM, entering the boot menu, and try starting in safe mode? Given that the in-place upgrade is an unsupported workflow, and given the [plethora ](https://social.technet.microsoft.com/Forums/en-US/2a85e26a-7414-449c-b1f9-8e9ad8bdeb2d/logonuiexe-hang-not-responsive-to-ctrlaltdel?forum=win10itprogeneral)[of ](https://social.technet.microsoft.com/Forums/en-US/2a85e26a-7414-449c-b1f9-8e9ad8bdeb2d/logonuiexe-hang-not-responsive-to-ctrlaltdel?forum=win10itprogeneral)[issues ]()people also have with Ctrl-Alt-Delete not responding at LoginUI.exe, we want to be sure that adding this feature will actually resolve the problem. 

I would also be interested to hear if this issue came up while going through a supported workflow - that may help prioritize this feature appropriately.

**Comment by lfernandes97** [comment]
@asinn826 

Not sure if this is still being looked at, but I was following https://learn.microsoft.com/en-us/azure/virtual-machines/windows-in-place-upgrade to upgrade my VM and I got stuck on the same issue as originally raised. I wasn't aware but I had to choose Desktop Experience option which was not mentioned on the workflow.
